packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/CSharpGen.cs

```diff
@@ -98,6 +98,8 @@ internal static string ParseOutputPath(string outputPath)
                 outputPath = Path.Combine(outputPath, "src");
             }
 
+            outputPath = Path.Combine(outputPath, GeneratedFolderName);
```

Comment:
Do we really need this method? It seems like it is only used for tests. Can we move it into the test class that needs it and make GeneratedFolderName internal?

Reply:
Nevermind, I see that this is the Generated Source output path. Can we rename the method to something like ParseGeneratedSourceOutputPath?